src/Connection.php

```diff
@@ -147,54 +147,212 @@ public function query(QueryBuilder $queryBuilder): PromiseInterface
      *
      * @return PromiseInterface
      */
-    public function queryBySQL(string $queryBuilder, array $parameters = []): PromiseInterface
+    public function queryBySQL(string $sql, array $parameters = []): PromiseInterface
     {
         return $this
             ->driver
-            ->query($queryBuilder, $parameters);
+            ->query($sql, $parameters);
     }
 
     /**
-     * Insert.
+     * Shortcuts.
+     */
+
+    /**
+     * Find one by id.
+     *
+     * connection->findOneById('table', ['id' => 1]);
+     *
+     * @param string $table
+     * @param array  $id
```

Comment:
I'd name it `$where` or `$conditions` since it doesn't necessarily have to be primary key (same for delete and update) and name the method simply `findOneBy`.